server/src/server/init-server.js

```diff
@@ -90,6 +91,7 @@ exports.init = async (knex, config) => {
     await server.register(Schmervice)
 
     await server.registerService(createEmailService)
```

Comment:
We might want to do the initialisation of these services in the `apiRoutes`-plugin. Then we wouldn't have to call `request.services(true)`, as it would be the same plugin that registered the schmervice-services. It can be argued that these services is not really a part of the api-routes though, so I've kept it in `init-server.js`.